docs/10-marketing/readme.md

```diff
@@ -0,0 +1,20 @@
+# Marketing Department
+
+Handbook for CivicActions' Marketing department
+
+## Lead
+
+Aaron Pava serves as our Chief Experience Officer and primary point of contact.
+
+## Continuous Learning / Guild
+
+Marketing has a weekly governance meeting that tensions can be raised that impact the Marketing Department or 
```

Comment:
Yeah - I would suggest that when existing teams are ready to morph into the Guild structure that they write this part of the page themselves.